util/internal/test/src/main/java/org/hibernate/search/util/impl/test/annotation/TestForIssue.java

```diff
@@ -18,11 +19,18 @@
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ ElementType.METHOD, ElementType.TYPE })
+@Repeatable(TestForIssue.List.class)
 public @interface TestForIssue {
 	/**
 	 * The key of a JIRA issue tested.
 	 *
 	 * @return The jira issue key
 	 */
 	String jiraKey();
```

Comment:
In another project (see https://github.com/hibernate/hibernate-ogm/blob/master/core/src/test/java/org/hibernate/ogm/utils/TestForIssue.java#L28), we made `jiraKey` an array.

Reply:
Right. I amended the commit to do the same.